articles/why-you-should-use-celery-with-rabbitmq/index.md

```diff
@@ -0,0 +1,463 @@
+            <h1 class="title">GenDog&#8482;</h1>
+            <p class="subtitle">Generate Dog Pictures By Breed</p>
+            <form method="POST">
+              <div class="field">
+                <label class="label">Pick A Breed</label>
+                <div class="control">
+                  <div class="select">
+                    <select name="breed" required>
+                      {% for breed in breeds %}
+                      <option value="{{breed}}">{{breed}}</option>
+                      {% endfor %}
+                    </select>
+                  </div>
+                </div>
+              </div>
+              <div class="field">
+                <label class="label">Limit</label>
+                <div class="control">
+                  <input
+                    class="input"
+                    type="number"
+                    style="width: 200px"
+                    name="limit"
+                    required
+                  />
+                </div>
+              </div>
+              <div class="field is-grouped">
+                <div class="control">
+                  <button
+                    class="button is-link"
+                    value="getDogPics"
+                    name="submit"
+                  >
+                    Submit
+                  </button>
+                </div>
+              </div>
+            </form>
+            <form method="POST">
+              <div class="control mt-4">
+                <button
+                  class="button is-link is-danger"
+                  value="clearDogPics"
+                  name="submit"
+                >
+                  Clear Photos
+                </button>
+              </div>
+            </form>
+          </div>
+        </div>
+        <div class="column">
+          {% if link|length > 0 %}
+          <ul>
+            {% for links in link %}
+            <li style="margin: 5px">
+              <img src="{{links}}" />
+            </li>
+            {% endfor %}
+          </ul>
+          {% endif %}
+        </div>
+      </div>
+    </section>
+  </body>
+</html>
+
+```
+
+Additional styling for the gallery done in our `main.css` file.
+
+```css
+ul {
+  display: flex;
+  flex-wrap: wrap;
+}
+li {
+  height: 40vh;
+  flex-grow: 1;
+}
+img {
+  max-height: 100%;
+  min-width: 100%;
+  object-fit: cover;
+  vertical-align: bottom;
+}
+li:last-child {
+  flex-grow: 10;
+}
+@media (max-aspect-ratio: 1/1) {
+  li {
+    height: 30vh;
+  }
+}
+@media (max-height: 480px) {
+  li {
+    height: 80vh;
+  }
+}
+@media (max-aspect-ratio: 1/1) and (max-width: 480px) {
+  ul {
+    flex-direction: row;
+  }
+  li {
+    height: auto;
+    width: 100%;
+  }
+  img {
+    width: 100%;
+    max-height: 75vh;
+    min-width: 0;
+  }
+}
+
+```
+
+![gendog1.png](/engineering-education/why-you-should-use-celery-with-rabbitmq/gendog1.png)
+
+
+![gendog2.png](/engineering-education/why-you-should-use-celery-with-rabbitmq/gendog2.png)
```

Comment:
Please provide a heading under the image, that describes what the image contains